Week1/varXInteger.js

```diff
@@ -0,0 +1,7 @@
+//giving an integer value to variable x
+let x;
+console.log("the value of my variable x will be: my favorite number");
```

Comment:
I think that you forgot to change this `my favorite number`, because that i not the value that `x` has.

Reply:
done.